src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -56,25 +58,6 @@ public MainWindow(PowerToysRunSettings settings, MainViewModel mainVM, Cancellat
             _settings = settings;
             InitializeComponent();
 
-            if (OSVersionHelper.IsWindows11())
-            {
-                WindowBackdropType = Wpf.Ui.Controls.WindowBackdropType.Acrylic;
-            }
-            else
-            {
-                WindowBackdropType = Wpf.Ui.Controls.WindowBackdropType.None;
-            }
```

Comment:
Isn't this still needed her in order not to crash on Win10? I see it's been added in SetTheme, but seems to run only conditionally.

Reply:
The code is now called in the `OnLoaded` event handler.
Got what you mean! Good catch!
I have pushed a commit😃

Reply:
Can you please verify if the fixed light/dark theme have transparency in Windows 11?

Reply:
> Can you please verify if the fixed light/dark theme have transparency in Windows 11?

Not sure what you mean here 😅 

Any specific app you want to check this in? I can send screenshots of it running on Windows 11.

Reply:
![image](https://github.com/microsoft/PowerToys/assets/26118718/d9909a43-eb33-4a9b-a592-905e9793a574)
![image](https://github.com/microsoft/PowerToys/assets/26118718/1e97aa22-efbb-466e-b4cb-1090a4b019f3)

Reply:
You mean this, right?
![image](https://github.com/microsoft/PowerToys/assets/26118718/9d9aa2f6-08a2-4561-b0ce-36304aef273e)

![image](https://github.com/microsoft/PowerToys/assets/26118718/f4283570-336b-4a56-9e38-ff4371cd440a)

That shadow of dark on top of light looks OK. Is there some other transparency I should be looking for? 🤔 Sorry, not the most attentive to visual details 😅